pkg/controllers/multicluster/serviceexport_controller.go

```diff
@@ -241,6 +258,12 @@ func (r *ServiceExportReconciler) extractEndpoints(ctx context.Context, svc *v1.
 					if version.GetVersion() != "" {
 						attributes[K8sVersionAttr] = version.PackageName + " " + version.GetVersion()
 					}
+
+					// Apply clusterID as endpoint attribute if it exists
+					if clusterId.Spec.Value != "" {
+						attributes[ClusterIdAttr] = clusterId.Spec.Value
```

Comment:
Have we considered adding this to our endpoint model as a field?

Reply:
In my opinion, I like it as an attribute. The other existing fields (Id, IP, Ports) seems like more intrinsic properties that would belong to an Endpoint. ClusterID seems more like an attribute or additional useful information to have when getting Endpoint information. Im happy with either. What do we think?

Reply:
Changed to field.

Reply:
As per @fredjywang 's design review meeting - confirmed to go with field. @fredjywang will also add new field for `ServiceType`